test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -312,7 +312,7 @@ const smsInstruction1 = 'We sent an SMS to +18008885555 with an Okta Verify setu
 const smsInstruction2 = 'Or try a different way to set up Okta Verify.';
 const emailInstruction1 = 'We sent an email to joy@okta.com with an Okta Verify setup link. To continue, open the link on your receiving device.';
 const emailInstruction2 = 'Or try a different way to set up Okta Verify.';
-const qrCodeInstruction1 = 'On your mobile device, download the Okta Verify app from the App Store (iPhone and iPad) or Google Play (Android devices).';
+const qrCodeInstruction1 = 'On your other device, download the Okta Verify app from the App Store® (iPhone® and iPad®) or on Google Play (Android® devices).';
```

Comment:
Does "Google Play" need to also have a trademark or registered mark?

Reply:
good question, ill check and make another PR if needed.